kmir/src/kmir/__main__.py

```diff
@@ -45,7 +46,8 @@ def __init__(self, input_file: Path, output_file: str | None, start_symbol: str)
 
 
 def _kmir_run(opts: RunOpts) -> None:
-    tools = semantics()
+    print(f'haskell_backend: {opts.haskell_backend}')
```

Comment:
This will probably go away before merging?
```suggestion
```

Reply:
Yep, that's some debugging left over.